openvdb/openvdb/python/pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[build-system]
+requires = [
+    "scikit_build_core==0.2.1",
```

Comment:
There is more recent releases. We could update to get new features (like editable installs)